Data/Defaults.lua

```diff
@@ -995,7 +1007,53 @@ Defaults.Layouts = {
         healthBarColorType = CUF.constants.UnitButtonColorType.CELL,
         healthLossColorType = CUF.constants.UnitButtonColorType.CUSTOM,
     },
-    boss = {
+}
+
+-- Focus was introduced with TBC
+if not CUF.vars.isVanilla then
+    Defaults.Layouts.focus = {
```

Comment:
You need to use the const to make the typing happy, it's a bit dumb xd
```suggestion
    Defaults.Layouts[CUF.constants.UNIT.FOCUS]
```